updateApi.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+BLUE='\033[1;34m'
+GREEN='\033[0;32m'
+RED='\033[0;31m'
+NC='\033[0m'
+BOLD='\033[1m'
+
+set -e
+
+CURRENT_DIR=$(pwd)
+BASE_DIR=$(cd "$(dirname "$0")" && pwd)
+API_PKG="github.com/devfile/api/v2"
+SCHEMA_URL="https://raw.githubusercontent.com/devfile/api/master/schemas/latest/devfile.json"
```

Comment:
I know its not frequent, but we may also want to hardcode and store 2.1.0 path since its never gonna change and there is a possibility that 2.1.0 schema maybe updated and in that case we can just pass input to this file to update the schema in our repo

Reply:
If we go down that path its probably easier to maintain the logic in go that bash, where the bash is just calls the go exec with the params

Reply:
2.1.0 path is the master one. Do you mean 2.0.0 path?

Reply:
yes sorry, 2.0.0 and so on when newer ones are out.

Reply:
store all info in `./updateApi.sh`, and allow to pass an extra argument to update schema for selected devfile version
(now only has case 2.0.0, 2.1.0 will be default case)